File: backend/javaserver/src/main/java/model/MetaEntry.java
package model;

public class MetaEntry {
	// Used for (de)serialization. Do not change.
	private String key;
	
	// Used for (de)serialization. Do not change.
	private String value;
	
	// Used for (de)serialization. Do not change.
	private String group;
	
	public MetaEntry() {
		
	}

	public String getKey() {
		return key;
	}

	public void setKey(String key) {
		this.key = key;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}

	public String getGroup() {
		return group;
	}

	public void setGroup(String group) {
		this.group = group;
	}
}

File: frontend/CommunicationLib/Model/DataModel/MetaEntry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommunicationLib.Model
{
    /// <summary>
    /// This class represents a MetaEntry for the MetaEntryList of an Item
    /// </summary>
    public class MetaEntry : RootElement
    {
        /// <summary>
        /// Used for (de)serialization. Do not change the property name.
        /// </summary>
        public string key { get { return id; } set { id = value; } }

        /// <summary>
        /// Used for (de)serialization. Do not change the property name.
        /// </summary>
        public string value { get { return _value; } set { _value = value; } }
        private string _value;

        /// <summary>
        /// Used for (de)serialization. Do not change the property name.
        /// </summary>
        public string group { get { return _group; } set { _group = value; } }
        private string _group;
    }
}